integr8ly/application-monitoring-operator
# v1.8.0

## Description
This pr updates AMO to 1.8.0. In this update we have updated the image used by prometheus, and fixed a bug relating to grafana not creating a service account when installed locally. 

## JIRA
https://issues.redhat.com/browse/MGDAPI-2372

## Verification
- Follow the steps in the updated README to install AMO locally.
- Verify prometheus operator is using the image `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:19ecf8f3cb62545bc81eeddd3ef8adc105ec5bad5538ee556aade7eb8f9d6dbf` 
- Verify all expected pods are created and working
